samples/charts/category-chart/axis-labels/App.razor

```diff
@@ -0,0 +1,47 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+    <div class="igOptions">
```

Comment:
add a slider with min = 0, max = 270 and step = 10 to control the XAxisLabel angle
add a slider with min = 0, max = 270 and step = 10 to control the YAxisLabel angle